fv3config/_config.py

```diff
@@ -33,6 +38,16 @@ def get_default_config():
     return config
 
 
+def get_n_processes(config_dict):
+    """Return the number of processes required to run the model based on the
+    configured processor layout.
+    """
```

Comment:
Public methods should have complete google-style doc strings (even if they're simple functions) so that their API is clear to the user.

Reply:
This *is* a google-style docstring. Under the style guide [Section 3.8.3](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#383-functions-and-methods), I think this counts as a case "where the function's name and signature are informative enough that it can be aptly described using a one-line docstring".

Reply:
Good point, I misread the style guide.